fair/src/autogluon/fair/learners/fair_frontier.py

```diff
@@ -0,0 +1,225 @@
+#TODO Either fully comment or remove this file.
```

Comment:
Is this TODO still valid? What is the status of this file? Should it be removed?

Reply:
This file allows optimization of arbitrary Scorables rather than GroupMetrics. But the code is much slower and less clean, and I'm not sure it's needed. 
We should discuss this. If there's a good use case it can stay.